installer/macosx/bundle_release.sh

```diff
@@ -176,22 +189,41 @@ echo copy readme.txt file
 cp ./readme.txt $BUILD_DIR/readme.txt
 
 if [ ! "$1" ]; then
+    set -x
+
+    echo "##teamcity[blockOpened name='Bundle code signing']"
+    echo
     echo Code signing...
-    ./codesign.mac.sh "$BUILD_DIR/Unipro UGENE.app"/Contents
+    echo pwd=`pwd`
+    echo ./codesign_clear-i386.sh "${TARGET_APP_DIR_RENAMED}"
+    bash ./codesign_clear-i386.sh "${TARGET_APP_DIR_RENAMED}"
+    echo ./codesign.mac.sh "${TARGET_APP_DIR_RENAMED}"
+    bash ./codesign.mac.sh "${TARGET_APP_DIR_RENAMED}"
+    echo "##teamcity[blockClosed name='Bundle code signing']"
+    
+    echo
+    echo Create pkg file
+    bash ./productbuild.sh \
+        "${TARGET_APP_DIR_RENAMED}" \
+        "ugene-${UGENE_VERSION}-mac-${ARCHITECTURE}-r${BUILD_VCS_NUMBER_new_trunk}".not-signed.pkg \
+        "ugene-${UGENE_VERSION}-mac-${ARCHITECTURE}-r${BUILD_VCS_NUMBER_new_trunk}".pkg
 
     echo
     echo Compressing symbols...
+    echo tar czf "${SYMBOLS_DIR}.tar.gz" "${SYMBOLS_DIR}"
     tar czf "${SYMBOLS_DIR}.tar.gz" "${SYMBOLS_DIR}"
 
     echo
     echo pkg-dmg running...
+    echo ./pkg-dmg --source $BUILD_DIR --target ugene-${UGENE_VERSION}-mac-${ARCHITECTURE}-r${BUILD_VCS_NUMBER_new_trunk}.dmg --license ./LICENSE.with_3rd_party --volname "Unipro UGENE $UGENE_VERSION" --symlink /Applications
     ./pkg-dmg --source $BUILD_DIR --target ugene-${UGENE_VERSION}-mac-${ARCHITECTURE}-r${BUILD_VCS_NUMBER_new_trunk}.dmg --license ./LICENSE.with_3rd_party --volname "Unipro UGENE $UGENE_VERSION" --symlink /Applications
 
     echo
     echo Signing dmg-file...
-    ./codesign.mac.sh \
-        ugene-${UGENE_VERSION}-mac-${ARCHITECTURE}-r${BUILD_VCS_NUMBER_new_trunk}.dmg \
-        "$BUILD_DIR/Unipro UGENE.app"/Contents/Info.plist
+    echo ./codesign.mac.sh ugene-${UGENE_VERSION}-mac-${ARCHITECTURE}-r${BUILD_VCS_NUMBER_new_trunk}.dmg
```

Comment:
Do we use pkg or dmg file? I see both of them, but, I think, that i's enough one only

Reply:
Not agree
pkg-file is installer
dmg-file contains pkg + Saples + ...